modules/travel_pay/app/services/travel_pay/client.rb

```diff
@@ -173,6 +148,41 @@ def veis_params
       }
     end
 
+    def request_ping(veis_token)
+      btsss_url = Settings.travel_pay.base_url
+      api_key = Settings.travel_pay.subscription_key
+
+      connection(server_url: btsss_url).get('api/v1/Sample/ping') do |req|
+        req.headers['Authorization'] = "Bearer #{veis_token}"
+        req.headers['Ocp-Apim-Subscription-Key'] = api_key
+        req.headers['X-Correlation-ID'] = SecureRandom.uuid
+      end
+    end
+
+    def request_authorized_ping(veis_token, btsss_token)
+      btsss_url = Settings.travel_pay.base_url
+      api_key = Settings.travel_pay.subscription_key
+
+      connection(server_url: btsss_url).get('api/v1/Sample/authorized-ping') do |req|
+        req.headers['Authorization'] = "Bearer #{veis_token}"
+        req.headers['BTSSS-Access-Token'] = btsss_token
+        req.headers['Ocp-Apim-Subscription-Key'] = api_key
+        req.headers['X-Correlation-ID'] = SecureRandom.uuid
+      end
+    end
+
+    def request_claims(veis_token, btsss_token)
+      btsss_url = Settings.travel_pay.base_url
+      api_key = Settings.travel_pay.subscription_key
+
+      connection(server_url: btsss_url).get('api/v1/claims') do |req|
+        req.headers['Authorization'] = "Bearer #{veis_token}"
+        req.headers['BTSSS-Access-Token'] = btsss_token
+        req.headers['Ocp-Apim-Subscription-Key'] = api_key
+        req.headers['X-Correlation-ID'] = SecureRandom.uuid
```

Comment:
Not for this PR unless you're feeling motivated, but at some point we should pull this header stuff into a separate method.

Reply:
I'm ok with doing this at a later time!

Reply:
It seems like the connection method could be pulled into a [configuration class](https://github.com/department-of-veterans-affairs/vets-api) as we see in many other of the service class patterns in Vets API unless theres some reason not to use this config. The service/config class pattern seems much cleaner

Reply:
I've had some problems with the Config class in the past, stemming from the fact that the Configuration base class is a singleton. We're choosing a simpler (more repeat-y) approach for now until we can figure out if the existing config pattern is going to work for us. Let us know if this blocks an approval and we can figure out a game plan.